target_chains/solana/pyth_solana_receiver_state/src/error.rs

```diff
@@ -0,0 +1,22 @@
+use anchor_lang::error_code;
+
+#[error_code]
+pub enum GetPriceError {
+    #[msg("Price Too Old")]
+    PriceTooOld = 10000, // Big number to avoid conflicts with other error codes
+    #[msg("Wrong Feed Id")]
+    WrongFeedId,
+    #[msg("Wrong Verification Level")]
+    WrongVerificationLevel,
```

Comment:
nitpick: I don't like using phrases like "wrong" "incorrect" etc in error names because they're not very precise. The error here is caused by the verification level in the account not being high enough for the requester. So is there a more precise way to say that? Maybe "InsufficientVerificationLevel"?

Reply:
same with the error above actually. `MismatchedFeedId` might be more precise